plugins/arangodbendorsements/arangodb.go

```diff
@@ -73,9 +73,9 @@ func checkEndPoint(ep string) error {
 func (e *ArangoStore) init(dbparams ArangoDBparams) error {
 	e.dbparams = dbparams
 	if dbparams.ConEndPoint == "" {
-		e.dbparams.ConEndPoint = dEndPoint
+		e.dbparams.ConEndPoint = defaultEP
 	} else if err := checkEndPoint(dbparams.ConEndPoint); err != nil {
-		return fmt.Errorf("init failed no valid connection endpoint: %w", err)
+		return fmt.Errorf("init failed, no valid connection endpoint: %w", err)
```

Comment:
maybe to increase the information content of the error message, you could add the URL that has failed validation